src/IO/ConvertLineReader.cpp

```diff
@@ -120,6 +120,10 @@ ConvertLineReader::ReadLine()
       return nullptr;
 
     /* fall through ... */
+    /* g++-8 of Armbian does not allow inplicit fallthrough */
```

Comment:
This has nothing to do with Armbian. There should be a generic solution for this problem.

Reply:
__attribute__ ((fallthrough)) is gcc specific. I wondered whether it is valid on other compilers. If not, I am glad to remove the comment and #if.

Reply:
I've added a commit adding a `gcc_fallthrough` macro:
https://github.com/XCSoar/XCSoar/commit/09571a4e2f822d2aa59f58206bca98332e0f3cef
You can use this one, but please make it a separate commit - it really has nothing to do with Armbian; just the Armbian compiler happens to be one more strict than other (maybe because it has other warning defaults).

Reply:
Will use.